homeassistant/components/rflink.py

```diff
@@ -317,6 +317,13 @@ def _async_handle_command(self, command, *args):
             cmd = str(int(args[0] / 17))
             self._state = True
 
+        elif command == 'toggle':
+            cmd = 'on'
+            if self._state is (STATE_UNKNOWN or False):
```

Comment:
I think this will give the behavior you seek:

    if self._state in [STATE_UNKOWN, False]:

Your code would first evaluate `(STATE_UNKNOWN or False)` (resulting in the string `unknown` and then compare if that is equal to `self._state`.

The `value in list` will check if the value equals one of the list items.